docs/admin/troubleshooting/cfr.md

```diff
@@ -0,0 +1,96 @@
+(cfr)=
+# CrateDB Flight Recorder (CFR)
+
+:::{rubric} About
+:::
+In a similar spirit like the [](#jfr), CFR helps to collect information about
+CrateDB clusters for support requests and self-service debugging.
+
+CFR is a utility application to acquire and export diagnostic information from
+CrateDB's [system tables](#systables) into an archive file. You can transmit
+this file to support engineers, in order to optimally convey relevant
+information about your cluster, mostly for debugging and troubleshooting
+purposes.
+
+:::{rubric} Details
+:::
+The CrateDB Flight Recorder (CFR) is an ETL application dumping all database
+tables in the `sys` schema into a timestamped tarball archive file.
+On the receiving end, the recording can be imported into another CrateDB
+instance, in order to inspect and analyze it.
+
+Flight recordings can be started against any running CrateDB cluster at runtime.
+The utility connects to CrateDB like a regular client, talking SQL.
+CFR is part of the CrateDB Toolkit (`ctk cfr`), and is also available as a
+standalone application `cratedb-cfr(.exe)`.
+
+
+## Synopsis
+
+:Export:
+
+    `cratedb-cfr sys-export` invokes the export operation.
+
+:Import:
+
+    `cratedb-cfr sys-import` invokes the import operation.
+
+
+## Install
+
+Select one of the standalone application bundles, matching the platform
+and architecture of the corresponding system where you intend to run CFR.
+
+::::{grid} 1 2 2 2
+
+:::{grid-item-card} {material-outlined}`download_for_offline;1.4em` Linux x64
+:link: https://github.com/crate-workbench/cratedb-toolkit/actions/runs/9826830191/artifacts/1674929097
```

Comment:
What is the approach to keep these links to release bundles up-to-date? I noticed it currently links to a specific GitHub Action run. Is there a possibility to have the artifacts as part of the regular release assets (https://github.com/crate-workbench/cratedb-toolkit/releases), so updating it here is just a matter of keeping it in sync with the latest cratedb-toolkit version number?

Reply:
Thanks, this is right on the spot. I've provided a relevant response citing you on the corresponding tracking ticket, so it will not get lost when merging this patch.

- https://github.com/crate-workbench/cratedb-toolkit/issues/160#issuecomment-2213691640